src/main/java/com/eliasnogueira/driver/BrowserFactory.java

```diff
@@ -133,4 +166,8 @@ public SafariOptions getOptions() {
      * @return a new AbstractDriverOptions instance based on the browser set
      */
     public abstract AbstractDriverOptions<?> getOptions();
+
+
```

Comment:
## Fix

Can you remove the empty line?

Reply:
done